tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -181,6 +181,7 @@ def generate_files(list, args):
     is_cpu_package = args.package_name in ['Microsoft.ML.OnnxRuntime', 'Microsoft.ML.OnnxRuntime.OpenMP']
     is_mklml_package = args.package_name == 'Microsoft.ML.OnnxRuntime.MKLML'
     is_cuda_gpu_package = args.package_name == 'Microsoft.ML.OnnxRuntime.Gpu'
+    is_tensorrt_package = args.package_name == 'Microsoft.ML.OnnxRuntime.TensorRT'
```

Comment:
Do we need a "Microsoft.ML.OnnxRuntime.TensorRT" package? Should trt in  "Microsoft.ML.OnnxRuntime.Gpu" ?

Reply:
I replied here:
https://github.com/microsoft/onnxruntime/pull/8687#discussion_r686967547

Actually the addition of _is_tensorrt_package_ has no effect on the process of creation of combined GPU package "Microsoft.ML.OnnxRuntime.Gpu".

Reply:
The reason I added this is that later if we want to solely build TRT EP nuget package with --use_tensorrt --build_nuget, we may need this modification.

Reply:
i think this may cause confusion.
if it's possible to create a GPU package with or without TensorRT then we are not doing the right thing.
likewise if we can create a TensorRT package which is the same as GPU package (with TensorRT), that is also not the right thing.
Let's make sure there's only one way to create a GPU package and it contains both CUDA and TensorRT EP's.

Reply:
> The reason I added this is that later if we want to solely build TRT EP nuget package with `./built.bat --config Release... --use_tensorrt ... --build_nuget ...`, we may need this modification.

is it still possible to name the package "Microsoft.ML.OnnxRuntime.Gpu" even if you create the package via that commandline?

Reply:
> > The reason I added this is that later if we want to solely build TRT EP nuget package with `./built.bat --config Release... --use_tensorrt ... --build_nuget ...`, we may need this modification.
> 
> is it still possible to name the package "Microsoft.ML.OnnxRuntime.Gpu" even if you create the package via that commandline?

I don't think you can name the package "Microsoft.ML.OnnxRuntime.Gpu" via this commandline which includes TensorRT EP libraries.

Reply:
To avoid confusion as Gorge mentioned, I'll remove this modification, aka adding package name "Microsoft.ML.OnnxRuntime.TensorRT".

We only want one GPU nuget package named "Microsoft.ML.OnnxRuntime.Gpu" which contains both CUDA and TensorRT EP's.